doc/wiki/3_iiQKA.OS2_RSI.md

```diff
@@ -0,0 +1,164 @@
+## iiQka driver (RSI)
+
+### Setup
+
+#### Client side
+It is recommended to use the driver on a real-time capable client machine (further information about setting up the PREEMPT_RT patch can be found [here](https://github.com/kroshu/kuka_drivers/wiki/6_Realtime)).
+
+To set up the controller with iiQWorks.Sim (which is necessary if RSI is not yet installed), a Windows machine is also required.
+
+##### Client IP configuration
+- Use the KRC's built in DHCP server (KSI) to connect to the windows machine (in this case leave the subnet settings as default) or set a fixed IP in the subnet of the KLI interface (in this case you can not have connection with both PCs). This is required to connect to iiQWorks.Sim and transfer the project to the KRC
```

Comment:
you can have connections with multiple PC-s using a switch, so I would delete that comment